client_test.go

```diff
@@ -0,0 +1,27 @@
+package symbiosis
+
+import (
+	// "reflect"
+	"github.com/go-resty/resty/v2"
+	"github.com/jarcoal/httpmock"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestNewClientEmptyValues(t *testing.T) {
+	_, err := NewClient("", "123")
```

Comment:
Are these tests using the correct NewClient? It looks like it takes two parameters with the first being the URL, but looking at the function it takes only the ApiKey.

Reply:
Actually already solved this, but didn't comm it it yet :) I indeed refactored it a bit